external-data/lib/TypeSerializer.ts

```diff
@@ -0,0 +1,151 @@
+export default class TypeSerializer {
```

Comment:
I can't quite figure out where this is getting used. Any thoughts on how we'll keep these in sync?

Reply:
It's used in TimeoutError which is used in Resolvable which is used in createPromise which is used in downloadFile.

Reply:
Removed